tests/test_widgets.py

```diff
@@ -24,13 +31,21 @@
             "MainFunctionGui",
             "show_file_dialog",
             "request_values",
+            "create_widget",
         )
     ],
 )
-def test_widgets(WidgetClass):
+def test_widgets(WidgetClass, backend):
     """Test that we can retrieve getters, setters, and signals for most Widgets."""
-    wdg: widgets.Widget = WidgetClass()
-    wdg.close()
+    use_app(backend)
+    try:
+        wdg: widgets.Widget = WidgetClass()
+        wdg.close()
+    except AttributeError as e:
+        if "Could not import object" in str(e):
+            pytest.skip(f"no {WidgetClass.__name__!r} in backend {backend!r}")
+            return
```

Comment:
this is a little too generic for me. Some widgets may stop working because of some import problems (something moved in Qt for example) and you will get only skip information instead of test faliture.